test/unit/specs/components/staking/__snapshots__/TabValidators.spec.js.snap

```diff
@@ -89,21 +94,26 @@ exports[`PageStaking has the expected html structure 1`] = `
       </div>
       </a>
    </li>
-   <li class=\\"li-validator li-validator-active li-validator-validator \\">
+   <li class=\\"li-validator li-validator-validator \\">
       <a href=\\"#/staking/validators/cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctqzh8yqw\\" class=\\"\\">
       <div class=\\"li-validator__value name\\">
          <span class=\\"validator-profile__status green\\"></span><img src=\\"~assets/images/validator-icon.svg\\" width=\\"48\\" height=\\"48\\" class=\\"avatar\\">
          <div class=\\"vert\\">
             <div class=\\"top\\">
                mr_mounty
             </div>
-      <div class=\\"bottom\\">
-         cosmosvaladdr…8yqw
+      <div class=\\"short-address\\">
+         <div id=\\"address\\" class=\\"address\\">
+            cosmosvalpub1234
+         </div>
+      <div class=\\"success\\">
```

Comment:
It is a little strange to have this DOM everywhere. How about using `v-if` instead of an `.active` class?